src/Storage/Storage/Storage.generated.format.ps1xml

```diff
@@ -332,8 +332,8 @@
         <TypeName>Microsoft.WindowsAzure.Commands.Common.Storage.ResourceModel.AzureStorageFileDirectory</TypeName>
       </ViewSelectedBy>
       <GroupBy>
-        <ScriptBlock>if ($_.ShareFileClient -eq $null) {$_.ShareDirectoryClient.ShareName} else {$_.ShareFileClient.ShareName}</ScriptBlock>
-        <Label>Share</Label>
+        <ScriptBlock>if ($_.ShareDirectoryClient -eq $null) {$_.ShareFileClient.AccountName + ", ShareName: " + $_.ShareFileClient.ShareName} else {$_.ShareDirectoryClient.AccountName + ", ShareName: " + $_.ShareDirectoryClient.ShareName}</ScriptBlock>
```

Comment:
Should any help example be updated for the output format change, like : https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage.Management/help/Rename-AzStorageFile.md#examples

Reply:
Updated.